sway-lib-std/src/token.sw

```diff
@@ -8,6 +8,8 @@ use ::identity::Identity;
 use ::revert::revert;
 use ::outputs::{Output, output_amount, output_count, output_type};
 
+const FAILED_TRANSFER_TO_OUTPUT_SIGNAL = 0xffff_ffff_ffff_0001;
```

Comment:
Looks reasonable, but what's the significance of the chosen hex value? Will this actually make it easier for devs to debug, or is this more for the SDK to decode and work with?

Reply:
Honestly 100% arbitrary. `0xffff_ffff_ffff_0000` just seemed like a reasonable _high_ offset for errors.

Reply:
This makes me think that the `Revert(42)` in `std::revert::require` would be a good candidate for a similarly-offset error value.

Reply:
True. Didn't want to touch it just yet because I don't know if it's been relied on anywhere right now. If not, we can certainly change it. I'll create an issue to clean up the errors and move them into a separate file or something.